upt_macports/templates/perl.Portfile

```diff
@@ -10,11 +10,24 @@ revision            0
 {% endblock %}
 
 {% block versions %}
+{%- if (pkg.upt_pkg.requirements.run|length > 0) or (pkg.upt_pkg.requirements.test|length > 0) %}
```

Comment:
Can't we write:

> {% if pkg.upt_pkg.requirements.run or pkg.upt_pkg.requirements.test %}

here?

Reply:
this changed was made, anything else here @Steap - otherwise just press "Resolve"